kokoro/linux/python/x86_64/build_python.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -eux
+#
+# This script builds the native extension or pure Python implementations
+# of Protocol Buffers.
+#
+# This script should be run from the root of the protobuf repo.
+
+# The following values can be overridden by environment variable:
+#
+# PROTOBUF_PYTHON_RUNTIME
+#   "cpp" (default, C++ native extension) or "python" (pure Python)
+: ${PROTOBUF_PYTHON_RUNTIME:=cpp}
+
+# Validate the runtime selection.
+if [[ ${PROTOBUF_PYTHON_RUNTIME} == "cpp" ]]; then
+  BUILD_NATIVE=1
+elif [[ ${PROTOBUF_PYTHON_RUNTIME} != "python" ]]; then
+  echo -e "Unrecognized PROTOBUF_PYTHON_RUNTIME: ${PROTOBUF_PYTHON_RUNTIME@Q}" >&2
+  exit 1
+fi
+
+# Builds the protobuf library. At a minimum, this is needed for protoc.
+build_libprotobuf() {
+  # To build the C++ extension, we must make sure to build PIC.
+  ./autogen.sh
```

Comment:
It is possible to use Bazel instead? I've been trying to reduce dependencies on Automake in hopes we can kill it eventually.